src/AutoRest.CSharp.V3/Plugins/Modeler.cs

```diff
@@ -115,5 +117,32 @@ public async Task<bool> Execute(IAutoRestInterface autoRest, CodeModel codeModel
 
             return true;
         }
+
+        private KnownMediaType[] GetMediaTypes(CodeModel codeModel)
```

Comment:
Add a comment for https://github.com/Azure/autorest.modelerfour/issues/103, so we can make sure to change it later.